introduction.md

```diff
@@ -5,3 +5,5 @@ Hello Sense,
 Mind if I keep my shoes on ?
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?